services/src/main/java/org/apache/druid/cli/FakeQuerySegmentWalker.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.druid.cli;
+
+import org.apache.druid.query.Query;
+import org.apache.druid.query.QueryRunner;
+import org.apache.druid.query.QuerySegmentWalker;
+import org.apache.druid.query.SegmentDescriptor;
+import org.joda.time.Interval;
+
+public class FakeQuerySegmentWalker implements QuerySegmentWalker
```

Comment:
nit: rename to NoopQuerySegmentWalker to be consistent with other druid naming conventions.